src/sonic-yang-models/yang-models/sonic-fips.yang

```diff
@@ -0,0 +1,36 @@
+module sonic-fips {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/sonic-fips";
+
+    prefix sonic-fips;
+
+    description "FIPS YANG Module for SONiC OS";
+
+    revision 2023-06-20 {
+        description "First Revision";
+    }
+
+    container sonic-fips {
+
+        container FIPS {
+
+            description "FIPS part of config_db.json";
+
+            container global {
+
+                leaf mode {
+                    type string {
+                        pattern '[\w]{1,255}';
```

Comment:
If the mode has a restricted range of acceptable values, we can create a pattern for it.
e.g. pattern "enforce|disable|none";

Reply:
Fixed, thanks.